<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
 <testsuite name="place move game command" timestamp="2016-12-17T17:21:21" hostname="localhost" time="0.008" errors="0" tests="2" skipped="0" disabled="0" failures="1">
  <testcase classname="place move game command" name="should emit MovePlaced on first game move" time="0.002" />
  <testcase classname="place move game command" name="should emit IllegalMove when square is already occupied" time="0.006">
   <failure type="exception" message="AssertionError: expected &apos;[{&quot;type&quot;:&quot;MovePlaced&quot;,&quot;user&quot;:{&quot;userName&quot;:&quot;Gummi&quot;},&quot;name&quot;:&quot;TheFirstGame&quot;,&quot;timeStamp&quot;:&quot;2014-12-02T11:29:29&quot;,&quot;side&quot;:&quot;X&quot;}]&apos; to be &apos;[{&quot;type&quot;:&quot;IllegalMove&quot;,&quot;user&quot;:{&quot;userName&quot;:&quot;Gummi&quot;},&quot;name&quot;:&quot;TheFirstGame&quot;,&quot;timeStamp&quot;:&quot;2014-12-02T11:29:29&quot;}]&apos;"><![CDATA[AssertionError: expected '[{"type":"MovePlaced","user":{"userName":"Gummi"},"name":"TheFirstGame","timeStamp":"2014-12-02T11:29:29","side":"X"}]' to be '[{"type":"IllegalMove","user":{"userName":"Gummi"},"name":"TheFirstGame","timeStamp":"2014-12-02T11:29:29"}]'
    at Assertion.fail (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/node_modules/should/cjs/should.js:205:17)
    at Assertion.value (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/node_modules/should/cjs/should.js:277:19)
    at /home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/server/socket-app/tictactoe/tictactoe-game.spec.js:199:53
    at Object.PlaceMove (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/server/socket-app/tictactoe/tictactoe-handler.js:46:25)
    at Object.executeCommand (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/server/socket-app/tictactoe/tictactoe-handler.js:64:38)
    at Object.<anonymous> (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/server/socket-app/tictactoe/tictactoe-game.spec.js:198:26)
    at attemptSync (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1950:24)
    at QueueRunner.run (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1938:9)
    at QueueRunner.execute (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1923:10)
    at Spec.queueRunnerFactory (/home/birkirfreyr/Desktop/3vikna/dt1/reference-tictactoe/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:714:35)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>